src/types/gecko-profile.js

```diff
@@ -426,6 +426,22 @@ export type LibMapping = {|
   codeId?: string, // e.g. "6132B96B70fd000"
 |};
 
+/**
+ * Log object that holds the profiling-related logging information for a
+ * single thread only. This is optional and older profiles don't have it.
+ */
+export type ThreadProfilingLog = {
+  [log: string]: mixed,
+};
+
+/**
+ * Log object that holds the profiling-related logging information.
+ * This is optional and older profiles don't have it.
+ */
+export type ProfilingLog = {
+  [tid: string]: ThreadProfilingLog,
```

Comment:
Nit: The key is really a `pid`, i.e., a Process ID and not a Thread ID. And the value type should be a `ProcessProfilingLog`.

Reply:
Oh, right thanks for correcting that, I'll change it.